guides/common/modules/proc_installing-amazon-ec2-plugin.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Installing_Amazon_EC2_Plugin_{context}"]
-= Installing Amazon EC2 Plugin
+= Installing Amazon EC2 plugin
```

Comment:
```suggestion
= Installing Amazon EC2 plug-in
```

Reply:
Won't change because the text contains 'plugin'. Please create a PR or issue to change. :-)